Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -364,6 +364,12 @@ May produce spurious warnings.
     "Include the Dafny runtime as source in the target language.") {
     IsHidden = true
   };
+  
+  // TODO is this a good message?
+  public static readonly Option<bool> VerifyInvariants = new("--verify-invariants", () => false,
+    "Enable verification of class invariants.") {
+    IsHidden = true
+  };
```

Comment:
I think it would make more sense to just have an `--invariants` flag to enable having invariants at all, as long as its clear that they aren't fully verified yet. It doesn't seem as useful to let anyone declare them but have them totally unchecked.